src/components/structures/TimelinePanel.tsx

```diff
@@ -1470,7 +1471,7 @@ class TimelinePanel extends React.Component<IProps, IState> {
             if (error.errcode == "M_FORBIDDEN") {
                 description = _t(
                     "Tried to load a specific point in this room's timeline, but you " +
-                        "do not have permission to view the message in question.",
+                    "do not have permission to view the message in question.",
```

Comment:
Normally, the code is already formatted by Prettier. Please do not change the formatting :)